src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IFixedStatement.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact]
+        [CompilerTrait(CompilerFeature.IOperation)]
+        public void FixedStatement_FixedClassVariableAndPrint()
+        {
+            string source = @"
+using System;
+
+class C
+{
+    private int i;
+
+    void M1()
+    {
+        unsafe
+        {
+            /*<bind>*/fixed(int *p = &i)
```

Comment:
>int *p = &i [](start = 28, length = 11)

It would be good to test a multiple declarator case.

Reply:
Added.